eos/b-decays/b-to-d-l-nu.cc

```diff
@@ -210,26 +343,30 @@ namespace eos
             Save<Parameter, double> save_m_l(_imp->m_l, _imp->parameters["mass::mu"]());
             Save<std::string> save_opt_l(_imp->opt_l._value, "mu");
 
-            br_muons = integrate<GSL::QAGS>(f, s_min_mu, s_max);
+            br_muons = integrate<GSL::QAGS>(f, s_min_mu, s_max_mu);
         }
 
         double br_taus;
         {
             Save<Parameter, double> save_m_l(_imp->m_l, _imp->parameters["mass::tau"]());
             Save<std::string> save_opt_l(_imp->opt_l._value, "tau");
 
-            br_taus = integrate<GSL::QAGS>(f, s_min_tau, s_max);
+            br_taus = integrate<GSL::QAGS>(f, s_min_tau, s_max_tau);
         }
 
         return br_taus / br_muons;
     }
 
     const std::string
     BToDLeptonNeutrino::description = "\
-The decay B->D l nu, where l=e,mu,tau is a lepton.";
+    The decay B->D l nu, where l=e,mu,tau is a lepton.";
 
     const std::string
     BToDLeptonNeutrino::kinematics_description_s = "\
-The invariant mass of the l-nubar pair in GeV^2.";
+    The invariant mass of the l-nubar pair in GeV^2.";
+
+    const std::string
+    BToDLeptonNeutrino::kinematics_description_c_theta_l = "\
+    The cosine of the charged lepton's helicity angle theta_l in the l-nubar rest frame.";
```

Comment:
Is it clear that "helicity angle" is meant w.r.t. the direction of the B-meson?
Maybe you can give more precise definition?

Reply:
I think it should be the direction of the D, not the B

Reply:
This should be fixed in a separate commit basically everywhere.

Reply:
@cBobeth @dvandyk thanks. but can you have a look at p.3 of https://arxiv.org/abs/1405.3719 and Fig.1,  where they define the angles (for B->D*lnu). there it says ".. theta_l is the polar angle between the charged lepton and the direction 'opposite' to D* meson in the l-nu rest frame..." . So why should it be different definition when considering the B->D case, or? i will have a look at other comments later, cheers

Reply:
@akokulu7 Christoph's point was, the the present definition of theta_l in the description is incomplete. And as I wrote: this is not a necessary change for this PR. @cBobeth can e.g. go over the descriptions of all angular distributions to fix it.

Reply:
As Danny says, the question was more or less, whether you would know what is meant by "helicity angle" - I would not understand it. So we have to check the literature for B->D,D* l v,
because I am just used to the definitions in B->K,K* ll. I will add a separate issue for this.